faceswap.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+import argparse
+from scripts.extract import ExtractTrainingData
+from scripts.train import TrainingProcessor
+from scripts.convert import ConvertImage
+
+parser = argparse.ArgumentParser()
+subparser = parser.add_subparsers()
+extract = ExtractTrainingData(subparser, "extract", "Extract the faces from a pictures.")
+convert = ConvertImage(subparser, "convert", "Convert a source image to a new one with the face swapped.")
+train = TrainingProcessor(subparser, "train", "This command trains the model for the two faces A and B.")
+arguments = parser.parse_args()
+arguments.func(arguments)
```

Comment:
I think it will be better to wrap everything into:
```
if __name__ == "__main__":
    ...
```

In order to not execute this by accidental importing faceswap as a module

Reply:
Oh yeah right. It's been quite some time since I've done some Python programming.